content/100.figure-table-legends.md

```diff
@@ -92,6 +92,24 @@ A value of 1 means that there is complete overlap between which cells are annota
 The heatmap shown is from library `SCPCL000498` [@doi:10.1016/j.devcel.2022.04.003].
 <br><br>
 
+<!-- Figure 5 -->
+![**Consensus cell type annotations in Brain and CNS tumors.**](https://raw.githubusercontent.com/AlexsLemonade/scpca-paper-figures/main/figures/compiled_figures/pngs/figure_5.png?sanitize=true){#fig:fig5 tag="5" width="7in"}
+
+A. Dot plot showing expression of cell-type-specific marker genes across all libraries from brain and central nervous system (CNS) tumors.
+Expression is shown for each broad consensus cell type annotation.
+The y-axis displays the consensus cell type observed across libraries, with the total number of cells indicated in parentheses.
+The x-axis displays marker genes, determined by `CellMarker2.0` [@doi:10.1093/nar/gkac947], used for consensus cell type validation for each cell types shown along the top annotation bar.
+Dots are colored by mean gene expression across libraries and sized proportionally to the percent of libraries they are observed in, out of all cells in brain and CNS tumor libraries.
```

Comment:
```suggestion
Dots are colored by mean gene expression across libraries and sized proportionally to the percent of libraries they are observed in, out of all cells with the same broad cell type annotation in brain and CNS tumor libraries.
```
Just to be clear its not proportional to all cells, just cells with the same cell type.